### PR TITLE
Feature/223 - Markdown linting implemented

### LINKED ISSUE #223: Implement markdown linting for the wiki
**Is your feature request related to a problem? Please describe.**
It can be easy to make mistakes when writing markdown, and not everyone follows the same standard.
Also it is easy to mistakenly create dead links by moving pages.

**Describe the solution you'd like**
For markdown linting (maybe also spelling?) to be implemented via GitHub Actions.
A possible tool for this could be <https://github.com/markdownlint/markdownlint> which supports custom styles.

**Describe alternatives you've considered**
Not using it, or using some other tool.

**Additional context**
Right now using raw markdownlint gives around 2500 errors, but many of them are Line Length, which probably should be disabled or modified to more than 80 characters.

### PR DESCRIPTION
**Pull request by Determinatoro**
fixes #223

### CODE REVIEW

**Comment by Determinatoro** [comment]
# Review Checklist 

@aau-giraf/team you have been assigned this. Here's your checklist.

Please use the following checklist:

## Code Documentation Only

- [x] It is documented which BLoCs/Screens the current class is related to.
- [x] The responsibility/functionality of the current class is explained. Also
  regarding the classes it is related to.
- [x] If any unusual decisions are introduced in the code, they reasoned about
  in the documentation.

## Wiki Related

- [x] There are no dead links or references.
- [x] The markdown renders correctly in MkDocs.
- [x] The content is placed in the correct page/folder/section.

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally).

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+jobs:
+  markdown:
+    runs-on: ubuntu-latest
+    steps:    
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Run mdl        
+        uses: actionshub/markdownlint@1.2.0   
+        with:  
+          path: './docs'
```
Comment (puchasa) [inline .github/workflows/ci.yml:13]: Jeg tænker stadig at det ville være en fordel at have en newline i slutningen af filerne. Det var denne her (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) forklaring på hvorfor vi bruger newlines jeg læste og jeg tænker at selvom det i det store og hele ikke betyder så meget kan det vel aldrig være forkert at følge standarden. Og så kan det godt være at linteren ikke læser de her filer men jeg tænker s t a d i g at det er en god ide at følge standarden.

File: run_mdl.bat [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@echo off
+SET REPOSITORY=%~dp0
```
Comment (zaztz) [inline run_mdl.bat:2]: test

**Review by puchasa — APPROVED** [review]

**Review by zaztz — APPROVED** [review]
Wiki Related

- [x]     There are no dead links or references
- [x]     The markdown renders correctly in MkDocs
- [x]     The content is placed in the correct page/folder/section

Everything looks fine

**Pull request merged**